randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -198,6 +218,8 @@ Extra - asset_id: collision_camera_004
   * Extra - actor_name: LE_ChozoUnlockAreaDNA
   * Extra - actor_type: weightactivatedplatform
   * Extra - start_point_actor_name: ST_ChozoUnlockAreaDNA_Checkpoint
+  > Pickup (Missile Tank)
+      Impossible
```

Comment:
Add TODO for later

Reply:
There is a TODO. the pretty-printer just removes it, but you can see it in the DB: 
![grafik](https://github.com/randovania/randovania/assets/38186597/56c87af3-618b-4440-a3cc-0c8eb516fc21)